src/Catty/Helpers/Util/TableUtil.m

```diff
@@ -44,8 +53,10 @@ + (CGFloat)heightForImageCell
 + (CGFloat)heightForCatrobatTableViewImageCell:(CGFloat)navBarHeight
 {
     CGFloat screenHeight = [Util screenHeight];
+       
+    screenHeight -= statusBar;
     screenHeight -= navBarHeight;
-    return screenHeight * 0.14f;
+    return screenHeight * 0.15f;
```

Comment:
To make this magic number more readable for future maintainers: we could get rid of this by calculating that automatically without using any magic number, e.g. by using something like `(100 - CONTINUE_CELL_RELATIVE_HEIGHT) / number of cells`. This will then also work in case another cell gets added.